python/plugins/db_manager/dlg_table_properties.py

```diff
@@ -50,6 +50,10 @@ def __init__(self, table, parent=None):
 
         self.db = self.table.database()
 
+        objClass = self.db.searchClass()
+        if objClass != "PGDatabase":
```

Comment:
Same as above, call a method that determines if there's support for comments in here (and don't explicitly check provider types)